connector-email/src/main/java/com/redhat/cloud/notifications/connector/email/processors/it/ITUserRequestPreparer.java

```diff
@@ -0,0 +1,54 @@
+package com.redhat.cloud.notifications.connector.email.processors.it;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.redhat.cloud.notifications.connector.email.constants.ExchangeProperty;
+import com.redhat.cloud.notifications.connector.email.model.settings.RecipientSettings;
+import com.redhat.cloud.notifications.connector.email.processors.it.pojo.request.ITUserRequest;
+import org.apache.camel.Exchange;
+import org.apache.camel.Processor;
+import org.apache.camel.component.http.HttpMethods;
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+@ApplicationScoped
+public class ITUserRequestPreparer implements Processor {
+
+    @ConfigProperty(name = "notifications.connector.user-provider.it.elements-per-page", defaultValue = "1000")
+    Integer maxElementsPage;
```

Comment:
Could you please create an `EmailConnectorConfig` class similar to [ServiceNowConnectorConfig](https://github.com/RedHatInsights/notifications-backend/blob/master/connector-servicenow/src/main/java/com/redhat/cloud/notifications/connector/servicenow/ServiceNowConnectorConfig.java), move all `@ConfigProperty` occurrences there and inject it everywhere you need to access config values?

Reply:
Good idea! Done!